site/content/en/docs/Advanced/multi-cluster-allocation.md

```diff
@@ -96,7 +96,7 @@ To enable multi-cluster allocation, set `multiClusterSetting.enabled` to `true`
 NAMESPACE=<namespace>
 FLEET_NAME=<fleet name>
 
-curl https://${EXTERNAL_IP}:443/v1alpha/gameserverallocation \
+curl --http2 https://${EXTERNAL_IP}:443/v1alpha/gameserverallocation \
```

Comment:
Don't forget to change the `v1alpha` to `v1alpha1` due to my mistake in PR #1361.

Reply:
Thanks! This has also more work to get the gRPC working.

Reply:
I couldn't reopen this PR again; so I opened a new PR: https://github.com/googleforgames/agones/pull/1365.
@drichardson thanks again for finding the error. Please let me know if you have any other feedback on the new PR.